certified-connectors/Cognito Forms/readme.md

```diff
@@ -0,0 +1,129 @@
+## Cognito Forms
+Cognito Forms lets you easily build powerful online forms, such as surveys, order forms, registration forms and more. For free.
+Please see our [support documentation](https://www.cognitoforms.com/support/63/data-integration/microsoft-power-automate) 
+or [contact us](https://www.cognitoforms.com/supportrequest/) for help with our public Power Automate connector.
+
```

Comment:
@gsmcdonald The publisher section is missing. Please refer the [template](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/readme.md).

Reply:
@sriyen-msft Do we also need to add the other sections in that template or just the publisher section?

Reply:
Also, who is the intended audience for the readme?

Reply:
We went ahead and added the other sections as well.